app/models/katello/content_view_version.rb

```diff
@@ -40,6 +39,7 @@ class ContentViewVersion < Katello::Model
              :class_name => "Katello::ContentViewVersionComponent"
     has_many :composites, :through => :content_view_version_composites, :source => :composite_version,
              :class_name => "Katello::ContentViewVersion", :inverse_of => :components
+    has_many :composite_content_views, through: :composites, source: :content_view
```

Comment:
I think this requires a bit of explanation. Take a look at our two join tables, which enable composite content views:

```rb
=> #<Katello::ContentViewComponent:0x000000001f8d4a90
 id: 1,
 composite_content_view_id: 4,
 created_at: Mon, 08 Oct 2018 14:23:59 UTC +00:00,
 updated_at: Thu, 11 Oct 2018 15:58:09 UTC +00:00,
 content_view_id: 2, # component content view ID
 latest: true,
 content_view_version_id: nil> # specific version, only non-nil if latest is false

=> #<Katello::ContentViewVersionComponent:0x000000001f64ec58
 id: 1,
 component_version_id: 6,
 composite_version_id: 7,
 created_at: Mon, 08 Oct 2018 14:24:03 UTC +00:00,
 updated_at: Mon, 08 Oct 2018 14:24:03 UTC +00:00>
```

The first table contains no publish information whatsoever. If you want to list composite content views containing a version X, you want to ask whether the version is published in the composite content view, or if it is just associated with the composite content view (and the next publish of the composite content view would result in the version being published in the composite content view).

Our API is confusing around this point. There should be a difference between a content view version's composite content views and its 'published in' composite content views.